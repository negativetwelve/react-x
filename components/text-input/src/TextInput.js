// React X
// NOTE(mark): Until we need custom functionality, we should assume react-native
// and react-native-web export a consistent TextInput.
export {default} from './TextInputX';
