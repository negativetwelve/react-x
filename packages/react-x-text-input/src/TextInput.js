// Libraries
import React from 'react';
import PropTypes from 'prop-types';

// React X
import TextInputX from './TextInputX';

// --------------------------------------------------
// Components
// --------------------------------------------------
const TextInput = (props) => <TextInputX {...props} />;

// --------------------------------------------------
// Props
// --------------------------------------------------
TextInput.propTypes = {
};

TextInput.defaultProps = {
};

export default TextInput;
