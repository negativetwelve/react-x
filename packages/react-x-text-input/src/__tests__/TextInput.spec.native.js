// Modules
import React from 'react';
import TextInput from '../TextInput';

/* eslint-disable no-undef */
describe('TextInput', () => {
  context('with no props', () => {
    itRenders(() => <TextInput />);
  });
});
