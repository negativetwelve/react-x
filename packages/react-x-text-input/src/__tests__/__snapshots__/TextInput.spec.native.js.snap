// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput with no props renders snapshot 1`] = `null`;
